File: becrowd/C/1009.c
#include <stdio.h>
int main() {
	char nomeVendedor[50];
	double salarioFixo, totalVendas, salarioFinal;
	scanf("%s", &nomeVendedor);
	scanf("%lf", &salarioFixo);
	scanf("%lf", &totalVendas);
	
	salarioFinal = salarioFixo + (totalVendas*0.15);
	printf("TOTAL = R$ %.2lf\n", salarioFinal);

	return 0;	
}

File: beecrowd/JavaScript/1009.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var nomeVendedor = parseFloat(lines.shift());
var salarioFixo = parseFloat(lines.shift());
var vendasMes = parseFloat(lines.shift());

var salarioFinal = salarioFixo + (vendasMes * 0.15);

console.log("TOTAL = R$ " + salarioFinal.toFixed(2));